web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/TrialsApiController.java

```diff
@@ -151,12 +151,30 @@ public ResponseEntity<Map<String, List<Trial>>> trialsGetByCancerTypes(
         return new ResponseEntity<>(result, status);
     }
 
-    private Tumor getTumor(JSONObject jsonObject, String oncoTreeCode) {
+    private Tumor getTumor(JSONObject jsonObjectOncotree, JSONObject jsonObjectTrials, String oncoTreeCode) {
         Tumor tumor = new Tumor();
-        if (jsonObject.containsKey(oncoTreeCode)) {
-            JSONObject tumorObj = (JSONObject) jsonObject.get(oncoTreeCode);
+        if (jsonObjectOncotree.containsKey(oncoTreeCode)) {
+            JSONObject tumorObj = (JSONObject) jsonObjectOncotree.get(oncoTreeCode);
             Gson gson = new Gson();
             tumor = gson.fromJson(tumorObj.toString(), Tumor.class);
+
+            List<Object> trials = (List<Object>) tumorObj.get("trials");
```

Comment:
What's the plan here if the obj does not have `trials`?

Reply:
Any comments on this?

Reply:
Currently if the oncotree code exists in the mapping, then there would not  be a case where the trials would be empty since the mapping is generated from the list of trials. If the query for the oncotree code is not available then an empty list would be returned